packages/lisk-transactions/src/transactions/1_register_second_passphrase.ts

```diff
@@ -330,4 +275,11 @@ export class SecondSignatureTransaction extends BaseTransaction {
 			state: { sender: strippedSender },
 		};
 	}
+
+	public sign(passphrase: string): void {
```

Comment:
do we need to set `_signature` to undefined if setting it again below

Reply:
I think we do because `getBytes` will take into account of the signature if exists

Reply:
Yeah I think we need it as in fact getBytes uses  `this._signature` in a ternary operation. Resolving.